extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -271,17 +287,29 @@ private function appendRecordsFromResponse( array &$records, $response, array &$
 				continue;
 			}
 
-			$wikiInfo = WikiFactory::getWikiByID($site);
+			if ( !isset($domainCache[$site]) ) {
```

Comment:
And to fix all these automatically, you can run [`stylize.php`](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#mediawiki-php-style-helper) on the file. It's available on your devbox.